app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import './cardStyle.scss';
+//import TitleDialog from './TitleDialog';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+    margin: 25,
+    height: 400,
+  },
+});
+
+export default function TitleCard({
+  title,
+  upcomingItem,
+  releaseDate,
+  description,
+  image,
+  rating,
+}) {
+  const classes = useStyles();
+
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  return (
+    <Card className={classes.root}>
+      <CardActionArea onClick={handleClickOpen}>
+        <CardMedia className='media' image={image} title='Movie poster' />
+        <CardContent>
+          <Typography
+            className='center text'
+            gutterBottom
+            variant='h5'
+            component='h2'
+          >
+            {title}
+          </Typography>
+          <Typography
+            className='inline text2'
+            variant='body1'
+            color='text'
+            component='p'
```

Comment:
Is there a reason to provide a specific parent component? The `variant` already provides `p`